torch/csrc/inductor/aoti_runtime/model.h

```diff
@@ -222,9 +222,12 @@ class AOTInductorModelBase {
       auto size = this->constant_shape(i);
       auto stride = this->constant_stride(i);
       auto offset = this->constant_offset(i);
+      auto layout = this->constant_layout(i);
+      auto opaque_metadata_ptr = this->opaque_metadata(i);
+      auto opaque_metadata_size = this->opaque_metadata_size(i);
 
       AtenTensorHandle tensor_handle;
-      AOTI_TORCH_ERROR_CODE_CHECK(aoti_torch_create_tensor_from_blob(
+      AOTI_TORCH_ERROR_CODE_CHECK(aoti_torch_create_tensor_from_blob_v2(
```

Comment:
Thanks for the suggestion. I've added a macro `AOTI_USE_CREATE_TENSOR_FROM_BLOB_V1` in the `codecache.py` file for the `is_fb_code()` branch. When this macro is defined, an assertion is added to require that `opaque_metadata_size` is 0 and then the V1 function will be called. When `opaque_metadata_size` is not 0, it's unsupported anyway before this PR. 

Due to this issue, I'll also skip the UTs which use MKLDNN opaque weight (`opaque_metadata_size !=0`) if `IS_FBCODE`. I suppose this could be removed once the fbcode upgrade the pytorch version used during runtime (like after a couple of weeks). Does this solution look good to you?

Reply:
SGTM. Thanks for a quick fix.